govc/USAGE.md

```diff
@@ -230,6 +230,12 @@ but appear via `govc $cmd -h`:
  - [namespace.cluster.enable](#namespaceclusterenable)
  - [namespace.cluster.ls](#namespaceclusterls)
  - [namespace.logs.download](#namespacelogsdownload)
+ - [namespace.supervisorsvc.activate](#namespacesupervisorsvcactivate)
```

Comment:
question: `supervisorsvc` or more user-friendly just `supervisor`?

Reply:
Hum, depends actually on the other APIs for a future:

I was trying to follow the API pattern for the service names (but without having something like `supervisorservice`, but per-api this endpoint is https://developer.vmware.com/apis/vsphere-automation/latest/vcenter/namespace_management/supervisor_services/

And then we probably will have in a future something like `supervisorsvc.cluster` to add/enable/disable per clusters (https://developer.vmware.com/apis/vsphere-automation/latest/vcenter/namespace_management/supervisor_services.cluster_supervisor_services/)

and finally the versions (`supervisorsvc.versions`).

So We may end up with a huge tree like:
- namespace.supervisorsvc.cluster.create
- namespace.supervisorsvc.version.rm

Don't have a strong opinion here, mostly thinking loud! Maybe namespace.service? (

Reply:
Yeah, `namespace.service` or even `namespace.svc`.

Reply:
Changed the command here to svc. 

I left the package name as supervisorsvc as this is the internal representation in the API, but the whole command now will be namespace.svc.